lib/flex_commerce_api/json_api_client_extension/requestor.rb

```diff
@@ -4,20 +4,20 @@ class Requestor < ::JsonApiClient::Query::Requestor
       # expects a record
       def create(record)
         request(:post, klass.path(record.attributes, record), {
-                         data: record.as_json_api
+                         body: { data: record.as_json_api }
```

Comment:
Some changes to bring us in line with interface changes within the `json_api_client` gem we're extending